asv/benchmarks/benchmark_mujoco.py

```
@@ -38,6 +38,72 @@ def _target_q(owner):
+    limit_hi = model.joint_limit_upper.numpy()
+    dof_total = limit_lo.shape[0]
+    coord_layout = init.shape[0] == model.joint_coord_count
+    quat_joints = (int(newton.JointType.FREE), int(newton.JointType.BALL), int(newton.JointType.DISTANCE))
+    for j, jt in enumerate(joint_types):
+        if jt in quat_joints:
+            continue
+        d0 = int(qd_starts[j])
+        d1 = int(qd_starts[j + 1]) if j + 1 < len(qd_starts) else dof_total
+        for d in range(d0, d1):
+            i = int(q_starts[j]) + (d - d0) if coord_layout else d
+            l = max(float(limit_lo[d]), -1.0) if limit_lo[d] > -1.0e6 else -1.0
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)